diff --git a/lib/unixTerminal.js b/lib/unixTerminal.js
--- a/lib/unixTerminal.js
+++ b/lib/unixTerminal.js
@@ -25,8 +25,8 @@
 var terminal_1 = require("./terminal");
 var utils_1 = require("./utils");
 var native = utils_1.loadNativeModule('pty');
 var pty = native.module;
-var helperPath = native.dir + '/spawn-helper';
+var helperPath = process.env.SYNAPSE_NODE_PTY_SPAWN_HELPER || native.dir + '/spawn-helper';
 helperPath = path.resolve(__dirname, helperPath);
 helperPath = helperPath.replace('app.asar', 'app.asar.unpacked');
 helperPath = helperPath.replace('node_modules.asar', 'node_modules.asar.unpacked');
